app/views/user_views.py

```diff
@@ -0,0 +1,65 @@
+from flask import Blueprint, jsonify, request, flash
+from flask_login import login_user, logout_user, login_required, current_user
+from ..db_services.user_service import get_all_users, create_user, get_user_by_id, update_user, delete_user, authenticate_user,get_user_tasks
+from ..route.tasks import user_blueprint
+from flask_jwt_extended import create_access_token
+
+@user_blueprint.route('/login', methods=['POST'])
+def login():
+    data = request.get_json()
+    email = data.get('email')
+    password = data.get('password')  
+    user = authenticate_user(email, password)
+    if user:
+        login_user(user)
+        access_token = create_access_token(identity=str(user.id))
+        return jsonify(message='Login successful', access_token=access_token)
+    else:
+        return jsonify(error='Invalid email or password')
+
+@user_blueprint.route('/logout')
+@login_required
+def logout():
+    logout_user()
+    flash('Logout successful', 'success')
+    return jsonify(message='Logout successful')
+
+@user_blueprint.route('/apis', methods=["GET", "POST"])
+def manage_users():
+    if request.method == "GET":
+        users_list = get_all_users()
```

Comment:
Only admin should get all the users, No other role should be able to do this.
Also create a separate route for the GET request here